client/asset/btc/spv.go

```diff
@@ -1160,12 +1195,17 @@ func (w *spvWallet) startWallet() error {
 		}
 	}
 	chainService, err := neutrino.NewChainService(neutrino.Config{
-		DataDir:          w.netDir,
-		Database:         w.neutrinoDB,
-		ChainParams:      *w.chainParams,
-		AddPeers:         addPeers,
-		ConnectPeers:     w.connectPeers,
-		BroadcastTimeout: 10 * time.Second,
+		DataDir:       w.netDir,
+		Database:      w.neutrinoDB,
+		ChainParams:   *w.chainParams,
+		PersistToDisk: true, // keep cfilter headers on disk for efficient rescanning
+		AddPeers:      addPeers,
+		ConnectPeers:  w.connectPeers,
+		// WARNING: PublishTransaction currently uses the entire duration
+		// because if an external bug, but even if the resolved, a typical
+		// inv/getdata round trip is ~4 seconds, so we set this so neutrino does
+		// not cancel queries too readily.
+		BroadcastTimeout: 6 * time.Second,
```

Comment:
Looks like the default is 5? So we're actually giving neutrino **more** time.

Reply:
Correct.  So 10 or even 30 would be fine with this approach, but I figured I would set it to something based on my empirical observations now that I've studies it more in practice.